docs/zh/latest/plugins/aws-lambda.md

```diff
@@ -0,0 +1,213 @@
+---
+title: aws-lambda
+keywords:
+  - APISIX
+  - Plugin
+  - AWS Lambda
+  - aws-lambda
+description: 本文介绍了关于 Apache APISIX aws-lambda 插件的基本信息。
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`aws-lambda` 插件用于将 [AWS Lambda](https://aws.amazon.com/lambda/) 作为动态上游集成至 `APISIX` ，将所有特定 `URI` 的请求代理到 `AWS` 云。
+
+插件启用后会终止对已配置 `URI` 的请求，并代表客户端向 `AWS Lambda Gateway URI` 发起一个新的请求，其中包括配置的授权详细信息、请求头、正文（三个参数都是从原始请求中传递的）。然后将带有头信息、状态代码和正文的响应返回给使用 `APISIX` 发起请求的客户端。
+
+本插件支持通过 `AWS API key` 和 `AWS IAM secrets` 进行授权。
+
+## 属性
+
+| Name                 | Type    | Required | Default | Valid values | Description                                                  |
```

Comment:
Please translate, refer to [plugins/fault-injection.md](https://github.com/apache/apisix/blob/master/docs/zh/latest/plugins/fault-injection.md?plain=1).